File: Java/Half_NForest.java
public class Solution {
    public static void nForest(int n) {
        // Write your code here

        for(int i=1; i<=n; i++){
            for(int j=0; j<i; j++){
                System.out.print("* ");

                if(j==i-1){
                    System.out.println("");
                }
            }
        }
    }
}
//https://www.naukri.com/code360/problems/n-2-forest_6570178?utm_source=youtube&utm_medium=affiliate&utm_campaign=striver_patternproblems&leftPanelTabValue=SUBMISSION

File: Python/Half_NForest.py
#https://www.naukri.com/code360/problems/n-2-forest_6570178?utm_source=youtube&utm_medium=affiliate&utm_campaign=striver_patternproblems&leftPanelTabValue=SUBMISSION
def nForest(n:int) ->None:
    # Write your solution here.

    for x in range(n+1):
        for y in range(x):
            print('* ', end='')
            if(y==x-1):
                print()

    pass
